platforms/core-runtime/client-services/build.gradle.kts

```diff
@@ -49,7 +51,6 @@ dependencies {
     // The client should not depend on core or core-api or projects that depend on these.
     // However, these project still contains some types that are shared between the client and daemon.
     api(projects.core)
-    api(projects.coreApi)
```

Comment:
🎉  (I see that the implementation dependency is still there, but this is an improvement regardless)